reflex/config.py

```diff
@@ -451,6 +451,9 @@ class Config:
     # The bun path
     bun_path: ExistingPath = constants.Bun.DEFAULT_PATH
 
+    # Timeout to do a production build of a frontend page.
```

Comment:
In the future should we just directly expose the next config?

Reply:
i feel that would get us even more attached to nextjs than we are right now